packages/build/src/steps/run_step.ts

```diff
@@ -114,8 +115,14 @@ export const runStep = async function ({
       return {}
     }
 
-    if (!quiet && !coreStepQuiet) {
-      logStepStart({ logs, event, packageName, coreStepDescription, error, netlifyConfig })
+    const logPluginStart = () => logStepStart({ logs, event, packageName, coreStepDescription, error, netlifyConfig })
+
+    let outputManager: OutputManager | undefined
+
+    if (coreStepId === undefined && featureFlags.netlify_build_reduced_output) {
```

Comment:
We don't want to use the output manager for core steps, since those run in the same process as Netlify Build and so piping stdout/stderr is not that simple. We can better control the output of core steps though, so that's okay.